spk/mosh/Makefile

```diff
@@ -0,0 +1,27 @@
+SPK_NAME = mosh
+SPK_VERS = 1.3.2
+SPK_REV = 1
+SPK_ICON = src/mosh.png
```

Comment:
I really love Org edition in Emacs tty over latency... but it definitely does not look like a product icon. If none better, simply remove it (probably you will face a packaging issue icon is required - but I have already fixed it - and I should submit a PR for it)